pkg/controller/vpc.go

```diff
@@ -317,9 +317,19 @@ func (c *Controller) handleAddOrUpdateVpc(key string) error {
 		return err
 	}
 
+	var externalSubnet *kubeovnv1.Subnet
+	isExternalSubnetExist := false
+	if c.config.EnableEipSnat && c.config.NetworkType != util.NetworkTypeVlan {
```

Comment:
`c.config.NetworkType` only influences the default subnet, the custom VPC will not influenced by this config

Reply:
the vlan e2e seems config EnableEipSnat as true when use vlan, so the e2e failed.
and also, the NetworkTypeVlan means not use vpc case , so custom vpc will not be used if the cluster use vlan default network.  if use custom vpc means the cluster not use vlan default network

Reply:
fixed